src/scripts/main.js

```diff
@@ -1,3 +1,55 @@
 'use strict';
 
 // write code here
+
+const li = document.getElementsByTagName('li');
+
+const list = [...li];
+
+function sortList(arr) {
+  const people = [];
+
+  for (const el of arr) {
+    el.dataset.name = el.innerText;
+    people.push(el.dataset);
+  }
+
+  const sorted = people.sort((a, b) => parseInt(b.salary
+    .replaceAll(',', '').replace('$', '')) - parseInt(a.salary
+    .replaceAll(',', '').replace('$', '')));
```

Comment:
you could remove `parseInt(b.salary
    .replaceAll(',', '')` into separate function and reuse